src/arch/utils/mainparam_utils.cpp

```diff
@@ -59,6 +62,7 @@ void listHelp() {
 #ifdef FORTE_TRACE_CTF
   printf("%-20s Set the output directory for TRACE_CTF\n", "  -t <directory>");
 #endif // FORTE_TRACE_CTF
+  printf("%-20s Set the timer to be used\n", "  -T TIMER");
```

Comment:
I think to be consistent this should be 

```suggestion
  printf("%-20s Set the timer to be used\n", "  -T <timer>");
```

but this was also missed for the FORTE_MULTIPLE_DEVICES option

Reply:
changed both